packages/sdk/src/index.ts

```diff
@@ -12,6 +7,8 @@ export {
 
 export { utf8ToBytes, bytesToUtf8 } from "@waku/utils/bytes";
 
+export * from "./content_topic.js";
```

Comment:
nit: I don't really think we need to export it

Reply:
I'd rather leave it in for now just because it breaks unit tests if these functions can't be accessed from tests package